languoids/tree/nucl1709/fini1244/fini1245/waru1269/nucl1777/mole1240/mole1239/md.ini

```diff
@@ -9,8 +9,8 @@ countries =
 	Papua New Guinea (PG)
 links = 
 	http://www.wikidata.org/entity/Q11936790
-longitude = -5.663114
-latitude = 146.525207
+latitude = -5.663114
+longitude = 146.525207
```

Comment:
Latitude and longitude must have been swapped on this one. Upon cursory inspection it didn't look like this was a systematic error, though.